impl/client/src/main/javascript/web/pentaho/data/Attribute.js

```diff
@@ -116,27 +116,50 @@ define([
        */
       this.type = type;
 
-      var isDiscrete = type !== "number" && type !== "date";
-      if(!isDiscrete && spec.isDiscrete != null)
-        isDiscrete = !!spec.isDiscrete;
+      var isContinuous;
+      if(type === "string" || type === "boolean") {
+        isContinuous = false;
+      } else if(spec.isContinuous == null) {
+        isContinuous = type === "number" || type === "date";
+      } else {
+        isContinuous = !!spec.isContinuous;
+      }
 
       /**
-       * Indicates if the attribute is considered discrete.
+       * Indicates if the attribute is considered **continuous**, or,
+       * on the other hand, **categorical**.
        *
        * When the attribute's type is one of
        * {@link pentaho.data.AtomicTypeName.STRING} or
        * {@link pentaho.data.AtomicTypeName.BOOLEAN}
-       * this property is always `true`.
+       * this property is **ignored** and the attribute is necessarily **categorical**.
```

Comment:
Ok, next time I will read the docs before the implementation. ;-)

My question above is answered: it was intentional. I'll just leave the comment to discuss the choice.